urdf2webots/writeRobot.py

```diff
@@ -239,15 +239,18 @@ def URDFBoundingObject(robotFile, link, level, boxCollision):
 
     for boundingObject in link.collision:
         initialIndent = boundingLevel * indent if hasGroup else ''
-        if not boxCollision and boundingObject.position != [0.0, 0.0, 0.0] or boundingObject.rotation[3] != 0.0:
+        if not boxCollision and (boundingObject.position != [0.0, 0.0, 0.0] or boundingObject.rotation[3] != 0.0 or boundingObject.scale != [1.0, 1.0, 1.0]):
             robotFile.write(initialIndent + 'Transform {\n')
             robotFile.write((boundingLevel + 1) * indent + 'translation %lf %lf %lf\n' % (boundingObject.position[0],
-                                                                                      boundingObject.position[1],
-                                                                                      boundingObject.position[2]))
+                                                                                        boundingObject.position[1],
+                                                                                        boundingObject.position[2]))
             robotFile.write((boundingLevel + 1) * indent + 'rotation %lf %lf %lf %lf\n' % (boundingObject.rotation[0],
-                                                                                       boundingObject.rotation[1],
-                                                                                       boundingObject.rotation[2],
-                                                                                       boundingObject.rotation[3]))
+                                                                                        boundingObject.rotation[1],
+                                                                                        boundingObject.rotation[2],
+                                                                                        boundingObject.rotation[3]))
+            robotFile.write((boundingLevel + 1) * indent + 'scale %lf %lf %lf\n' % (boundingObject.scale[0],
+                                                                                    boundingObject.scale[1],
+                                                                                    boundingObject.scale[2]))
```

Comment:
I have a general remark here: if the value to be written for a translation, rotation or scale (or any other field) is the default value, then we shouldn't write it. That would make the proto output much cleaner and more readable.

Reply:
Totally agree with that. I will do it here.